Venmo/src/main/java/com/braintreepayments/api/VenmoClient.java

```diff
@@ -80,25 +80,26 @@ public void showVenmoInGooglePlayStore(@NonNull FragmentActivity activity) {
     public void createPaymentAuthRequest(@NonNull final FragmentActivity activity,
                                          @NonNull final VenmoRequest request,
                                          @NonNull VenmoPaymentAuthRequestCallback callback) {
-        braintreeClient.sendAnalyticsEvent("pay-with-venmo.selected");
+        braintreeClient.sendAnalyticsEvent(VenmoAnalytics.TOKENIZE_STARTED.event);
         braintreeClient.getConfiguration((configuration, error) -> {
             if (configuration == null && error != null) {
                 callback.onVenmoPaymentAuthRequest(new VenmoPaymentAuthRequest.Failure(error));
-                braintreeClient.sendAnalyticsEvent("pay-with-venmo.app-switch.failed");
+                braintreeClient.sendAnalyticsEvent(VenmoAnalytics.TOKENIZE_FAILED.event);
```

Comment:
We used "app-switch" failed more previously, but to be more accurate, I updated this to only be "app switch failed" for errors returned from actual app switching, or if Venmo is not installed.